tests/functional/i/isinstance_second_argument_py310.rc

```diff
@@ -0,0 +1,2 @@
+[testoptions]
+min_pyver=3.10
```

Comment:
maybe you could allow the test to run also for earlier versions like was done in `redundant-typehint-argument`:
```suggestion
[testoptions]
min_pyver=3.7

[TYPING]
runtime-typing=no
```

Reply:
What does the `runtime-typing` option do? Also, is there a need to specify 3.7 as the min version, since that's already the min version being used?

Reply:
Now that I think about it, this check is different because it doesn't deal with type annotations. In this case the types involved will actually be called, so I think this option won't apply.

Reply:
you don't have to explicitly write `min_pyver=3.7` but it would be just clearer for the test reader to understand why `runtime-typing` was used. There is some documentation `runtime-typing` at `pylint/extensions/typing.py`. It will identify type annotations if it can for python versions 3.7-3.9 with the `|` operation.

> Now that I think about it, this check is different because it doesn't deal with type annotations. In this case the types involved will actually be called, so I think this option won't apply.

I think you are right. These are not annotations it the actual code. so if someone uses the `|` operation they can only do that in py3.10+.